### PR TITLE
Update individual PPP for week 12

### PR DESCRIPTION
**Pull request by nealetham**
To update your own PPP with your contributions:
- [ ] Jordan
- [ ] Naren
- [ ] Alvin
- [x] Neale
- [x] Nicholas

### CODE REVIEW

File: docs/team/nseah21.md [hunk 1]
```diff
@@ -1,87 +1,63 @@
...
+RC4HDB is a housing database that offers a convenient and intuitive interface for housing management staff to streamline their daily operations. 
 
-### Code contributed:
+Given below are my contributions to the project.
 
-I contributed code to the following modules/classes:
-1. *Coming soon*
+* **New Feature**: Added the ability to show/hide table columns. 
+    * What it does: allows the user to hide any number of unwanted columns in the table. Columns that have been hidden can be restored by using the reset command.
+    * Justification: This feature improves the product significantly because a user can de-clutter their screen and only focus on the information that they require, saving them time and effort in their work.
```
Comment (nealetham) [inline docs/team/nseah21.md:14]: This is up to your own discretion, but I would not really recommend putting the justification here. Maybe you could forward them to the portion in the DG? or some other place. This is just my own opinion, it's also fine to keep it here.

File: docs/team/nseah21.md [hunk 2]
```diff
@@ -1,87 +1,63 @@
...
+    * Highlights: This enhancement affects the UI directly and had to be implemented carefully to minimise coupling between components. I was able to apply the observer pattern in developing this feature.
 
-You may view these contributions in more detail at [this link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=nseah21&breakdown=true).
+* **New Feature**: Added the venue booking feature that allows the user to manage recurrent venue bookings.
+    * Justification: This feature improves the product significantly as low-level operations such as clearing expired bookings and checking for booking clashes are handled for the user by RC4HDB.
```
Comment (nealetham) [inline docs/team/nseah21.md:18]: Same as above.

File: docs/team/nseah21.md [hunk 3]
```diff
@@ -1,87 +1,63 @@
...
 
-I contributed to the following sections of the user guide:
-1. [Locating residents by name : `find`](https://ay2223s1-cs2103t-w12-3.github.io/tp/UserGuide.html#locating-residents-by-name--find)
-2. [Filtering residents by field : `filter`](https://ay2223s1-cs2103t-w12-3.github.io/tp/UserGuide.html#filtering-residents-by-field--filter)
+* **Community**:
+    * PRs reviewed (with non-trivial review comments): [\#30](https://github.com/AY2223S1-CS2103T-W12-3/tp/pull/30), [\#55](https://github.com/AY2223S1-CS2103T-W12-3/tp/pull/55), [\#57](https://github.com/AY2223S1-CS2103T-W12-3/tp/pull/57), [\#72](https://github.com/AY2223S1-CS2103T-W12-3/tp/pull/72), [\#73](https://github.com/AY2223S1-CS2103T-W12-3/tp/pull/73), [\#98](https://github.com/AY2223S1-CS2103T-W12-3/tp/pull/98), [\#114](https://github.com/AY2223S1-CS2103T-W12-3/tp/pull/114), [\#122](https://github.com/AY2223S1-CS2103T-W12-3/tp/pull/122), [\#132](https://github.com/AY2223S1-CS2103T-W12-3/tp/pull/132), [\#182](https://github.com/AY2223S1-CS2103T-W12-3/tp/pull/182)
```
Comment (nealetham) [inline docs/team/nseah21.md:45]: I think it's good to show that you have reviewed a lot of pull requests, but you could also only list the more notable ones, just like how I did it.

File: docs/team/nseah21.md [hunk 4]
```diff
@@ -1,87 +1,63 @@
...
+    * Participated in forum discussions (examples: [\#383](https://github.com/nus-cs2103-AY2223S1/forum/issues/383), [\#401](https://github.com/nus-cs2103-AY2223S1/forum/issues/401))
+    * Reported bugs and suggestions for other teams in the class (examples: [\#310](https://github.com/AY2223S1-CS2103T-T12-4/tp/issues/310), [\#315](https://github.com/AY2223S1-CS2103T-T12-4/tp/issues/315), [\#328](https://github.com/AY2223S1-CS2103T-T12-4/tp/issues/328), [\#332](https://github.com/AY2223S1-CS2103T-T12-4/tp/issues/332), [\#362](https://github.com/AY2223S1-CS2103T-T12-4/tp/issues/362))
```
Comment (nealetham) [inline docs/team/nseah21.md:47]: Same as above.

**Review by nealetham — COMMENTED** [review]
Looks good Nicholas!

File: docs/team/nealetham.md [hunk 5]
```diff
@@ -1,156 +1,52 @@
...
+
+### Community
+* Reviewed [***27*** pull requests](https://github.com/AY2223S1-CS2103T-W12-3/tp/pulls?q=is%3Apr+is%3Aclosed+reviewed-by%3A%40me). Many had non-trivial review comments, such as [#110](https://github.com/AY2223S1-CS2103T-W12-3/tp/pull/110), [#257](https://github.com/AY2223S1-CS2103T-W12-3/tp/pull/257) & [#76](https://github.com/AY2223S1-CS2103T-W12-3/tp/pull/110). Discussions also took place offline.
+* Contributed to forum discussions, such as [differences between association, composition, aggregation](https://github.com/nus-cs2103-AY2223S1/forum/issues/86#issuecomment-1229400456) & [failing Codecov/patch check](https://github.com/nus-cs2103-AY2223S1/forum/issues/330).
+* Surface [***12*** potential bugs & suggestions](https://github.com/nealetham/ped/issues?q=is%3Aopen+is%3Aissue) for other teams during PE-D. These include [non-functioning theme switcher](https://github.com/nealetham/ped/issues/4) & [missing target-user & value proposition](https://github.com/nealetham/ped/issues/10).
```
Comment (nseah21) [inline docs/team/nealetham.md:40]: This is a good way of organising your community contributions. 😄

**Pull request merged**